tests/functional/job-file-trap/00-sigusr1.t

```diff
@@ -20,6 +20,8 @@
 # will no longer be poll-able after the kill.
 . "$(dirname "$0")/test_header"
 
+skip_all "TODO decide whether to re-instate this"
```

Comment:
Is this to be decided at a later date or a reminder for this pr?

Reply:
Laters.